sourcemash/api/feeds.py

```diff
@@ -81,6 +88,13 @@ def post(self):
             db.session.add(feed)
             db.session.commit()
 
+            # Scrape feed (but don't fail if redis-server is down)
+            try:
+                q = Queue('default', connection=conn)
+                job = q.enqueue_call(func=scrape_articles_only, args=(feed.id,), at_front=True)
+            except:
+                pass
```

Comment:
So what happens if the redis server is down? We just wait until the major scrape?